coretex/cli/commands/node.py

```diff
@@ -94,21 +87,75 @@ def update() -> None:
         ):
             return
 
-    click.echo("Stopping Coretex Node...")
     node_module.stop()
-    click.echo("Successfully stopped Coretex Node.")
 
-    click.echo("Starting Coretex Node...")
     node_module.start(f"{repository}:{tag}", config)
-    click.echo("Successfully started Coretex Node.")
+
+
+@click.command()
+@click.option("--verbose", is_flag = True, help = "Configure node settings manually.")
+def config(verbose: bool) -> None:
+    if node_module.isRunning():
+        if click.prompt("Node is already running. Do you wish to stop the Node? (Y/n)",
+            type = bool,
+            default = True,
+            show_default = False):
+            node_module.stop()
+
+        click.echo("If you wish to reconfigure your node, use coretex node stop commands first.")
+        return
+
+    config = loadConfig()
+
+    if isNodeConfigured(config):
+        if not click.prompt(
+            "Node configuration already exists. Would you like to update? (Y/n)",
+            type = bool,
+            default = True,
+            show_default = False
+        ):
+            return
+
+    click.echo("[Node Configuration]")
+
+    config["storagepath"] = Path.home() / ".coretex"
+    config["nodeName"] = click.prompt("Node name", type = str)
+    config["nodeAccessToken"] = node_module.registerNode(config["nodeName"])
+
+    if isGPUAvailable():
+        isGPU = click.prompt("Would you like to allow access to GPU on your node? (Y/n)", type = bool, default = True)
+        config["image"] = "gpu" if isGPU else "cpu"
+    else:
+        config["image"] = "cpu"
+
+    if not verbose:
+        config["nodeRam"] = node_module.DEFAULT_RAM_MEMORY
+        config["nodeSwap"] = node_module.DEFAULT_SWAP_MEMORY
+        config["nodeSharedMemory"] = node_module.DEFAULT_SHARED_MEMORY
+
+        click.echo("To configure node manually run coretex node config with --verbose flag.")
+    else:
+        config["storagepath"] = click.prompt("Storage path (press enter to use default)", Path.home() / ".coretex", type = str)
+        config["nodeRam"] = click.prompt("Node RAM memory limit in GB (press enter to use default)", type = int, default = getAvailableRamMemory())
```

Comment:
Why not use `default = node_module.DEFAULT_RAM_MEMORY` here for those default values?